### ISSUE TITLE
Open Sourcing? 

### ISSUE THREAD

**Issue by TheDrawingCoder-Gamer** [ephtracy/ephtracy.github.io#243]
MagicaVoxel isn't a commercial project, and is a very useful program. Why not open source it? Are there plans to make it commercial/make a pro version? Licenses usually can handle most things (except properitery). MagicaVoxel is also much better than other FOSS alternatives.

**Comment by TheDrawingCoder-Gamer** [comment]
This could also apply to other projects

**Comment by Valoneu** [comment]
i hope it will never be payed

**Comment by TheDrawingCoder-Gamer** [comment]
Yeah it's currently free but not libre, and it confuses me sometimes when something is free as in price but not free as in freedom

**Comment by tigercoding56** [comment]
> freeware is not free, nor is the freedom

depends on your definition of "free" 

which in my case is :+1: 
freedom to do whatever
which 
depending on your country
is free   as in cost

**Comment by randomairborne** [comment]
Could you explain what you mean by "freeware is not free, nor is the freedom"? Does that mean that it costs money to develop? That lawsuits are expensive with open-source?
